Solutions/Corvus.Leasing.Abstractions/Corvus/Leasing/Exceptions/LeaseAcquisitionUnsuccessfulException.cs

```diff
@@ -61,6 +61,6 @@ protected LeaseAcquisitionUnsuccessfulException(System.Runtime.Serialization.Ser
         /// <summary>
         /// Gets the lease policy used during the attempt to acquire the lease.
         /// </summary>
-        public LeasePolicy LeasePolicy { get; }
+        public LeasePolicy? LeasePolicy { get; }
```

Comment:
I'm not convinced this should be nullable. I suspect it's only because of the other "standard" exception constructors, none of which is actually used.

If you remove all the ctors we're not using (which is what I did recently on another Corvus project where this exact same issue came up) I think you'll find you'll be able to make this non-nullable, which would be more useful.